examples/apps/generic_client/generic_client.py

```diff
@@ -176,7 +176,7 @@ def Main(args=None):
     # Connect to registry list and retrieve registry
     global uri
     if not uri and mode == "listing":
-        registry_list_instance = direct_jrpc.create_worker_registry_list(
+        registry_list_instance = direct_jrpc.get_worker_registry_list_instance(
             config
```

Comment:
Do we need to pass config parameter again in this API since we are already passing this parameter in constructor ?

Reply:
Not required